services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileOnDisk.java

```diff
@@ -64,4 +68,8 @@ public void setBytes(byte[] bytes) {
   public void prepare(ImmutableStack<Object> indices) {
     // Method override exists here to comply with the implementation rules for the Writable interface.
   }
+
+  public byte[] getBytesForChecksum() {
```

Comment:
never called

Reply:
It is - and it is also overwritte in ArchiveOnDisk. So lets make it protected.